src/AggregateFunctions/AggregateFunctionGroupSortedArrayData.h

```diff
@@ -0,0 +1,174 @@
+#pragma once
+
+#include <IO/ReadBuffer.h>
+#include <IO/ReadHelpers.h>
+#include <IO/VarInt.h>
+#include <IO/WriteBuffer.h>
+#include <IO/WriteHelpers.h>
+
```

Comment:
This file contains the code for the internal data set I am using.
It may use single value implemented using multiset or a pair (value and weigth) implemented using multimap.
It also support different value types so, to just provide one final class I am doing some complex staff with templates and inheritance here.